include/implGBL/EUTelNav.h

```diff
@@ -0,0 +1,93 @@
+/** Basic navigation tools for general track fitting.  
```

Comment:
This will only read/write the final tracks/Hits in the LCIO format consistent with EUTelTrack EUTelState EUTelHIt. This can be updated to change to the usual format but for now I thought best to keep separate.